### ISSUE TITLE
Docker fails on command-lines longer than 65535 characters

### ISSUE THREAD

**Issue by remram44** [ViDA-NYU/reprozip#111]
This is annoying because right now the files to be unpacked are listed on the command-line for `tar`.
```
 ---> Running in 8767e51f7275
/bin/sh: 1: Syntax error: end of file unexpected
```

**Comment by remram44** [comment]
Vagrant also lists paths on the tar command-line (in setup.sh) if using `--dont-use-chroot`, this might cause a problem as well.

**Comment by remram44** [comment]
tar has `-T` (`--files-from`), which should be available everywhere. Should probably use that.

**Comment by remram44** [comment]
Addressed by 83283885.
